calm/flow-example/account-creation/account-creation-flow.json

```
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-09/meta/flow.json",
+  "flows": {
+    "flows": [
+      {
+        "flowUniqueId": "account-creation-flow-001",
+        "name": "Account Creation Flow",
+        "description": "This flow outlines the steps for a bank user to create a new bank account using the web client.",
+        "flow-requirement-url": "https://calm.finos.org/draft/2024-09/meta/flow-requirement.json",
+        "flow-requirement-instance": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/flow-example/account-creation/account-creation-requirement.json",
```

You don't need to specify the json schema here as it will be inferred because it should be specified as the schema of the requirement instance.

```suggestion
        "flow-requirement-instance": "https://calm.finos.org/flow-example/account-creation/account-creation-requirement.json"
```